### PR TITLE
test(IDX-3144): drop dependency on //rs/tests from //rs/tests/testing_verification:...

### PR DESCRIPTION
**Pull request by basvandijk**
As part of IDX-3144 this drops the dependency on the `//rs/tests:tests` library from the system-tests in `//rs/tests/testing_verification:...` by either inlining the tests or just dropping the dependency since it was not actually used.

### CODE REVIEW

File: rs/tests/testing_verification/ii_delegation_test.rs [hunk 1]
```diff
@@ -14,3 +28,142 @@ fn main() -> Result<()> {
 
     Ok(())
 }
+
+pub fn config(env: TestEnv) {
```
Comment (cgundy) [inline rs/tests/testing_verification/ii_delegation_test.rs:32]: This seems to be introducing a lot of new code - is this also part of dropping dependencies? Or can this be a separate PR?

Comment (basvandijk) [reply rs/tests/testing_verification/ii_delegation_test.rs:32]: No, this just inlines the code from `rs/tests/src/ii_delegation_test.rs` into the binary.

Comment (cgundy) [reply rs/tests/testing_verification/ii_delegation_test.rs:32]: Oh okay yeah now I see that `rs/tests/src/ii_delegation_test.rs` was deleted. Makes sense. 👍

**Review by cgundy — APPROVED** [review]
